django/http/response.py

```diff
@@ -299,10 +299,15 @@ def __init__(self, content=b'', *args, **kwargs):
         self.content = content
 
     def __repr__(self):
-        return '<%(cls)s status_code=%(status_code)d, "%(content_type)s">' % {
+        if 'Content-Type' in self:
+            return '<%(cls)s status_code=%(status_code)d, "%(content_type)s">' % {
+                'cls': self.__class__.__name__,
+                'status_code': self.status_code,
+                'content_type': self['Content-Type'],
+            }
+        return '<%(cls)s status_code=%(status_code)d>' % {
```

Comment:
It would be less duplication to do something like:

```python
if 'content-type' in self:
    content_type_bit = ', "{}"'.format(self['content-type'])
else:
    content_type_bit = ''
return '<{cls} status_code={status_code}{content_type_bit}>'.format(
    ...
```

ditto below

I think moving to `str.format` is better but idk Django's current policy since there's a lot of `%` formatting around still.